### ISSUE TITLE
gnt-instance list -F 'status = "running"' returns ADMIN_down instances

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1156]
Originally reported of Google Code with ID 1100.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

$ sudo gnt-cluster --version
gnt-cluster (ganeti v2.14.0) 2.14.0

$ sudo gnt-cluster version
Software version: 2.14.0
Internode protocol: 2140000
Configuration format: 2140000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version v2.14.0

$ sudo hspace --version
hspace (ganeti) version v2.14.0
compiled with ghc 7.6
running on linux x86_64

<b>What distribution are you using?</b>
Ubuntu 12.04

<b>What steps will reproduce the problem?</b>
1. Have instances in an ADMIN_down state.
2. Have instances in a running state.
3. # gnt-instance list -F 'status == "running"'

<b>What is the expected output? What do you see instead?</b>
Both instances which are running as well as instances which are ADMIN_down are returned in the output. I would expect that only instances which are running would be returned in the output.

<b>Please provide any additional information below.</b>

```
Originally added on 2015-06-08 20:39:02 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Reproduced the described behavior.
```
Originally added on 2015-06-18 12:14:36 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
I am working on this.
```
Originally added on 2015-06-18 13:05:27 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This issue is fixed in 2.12 with the following patches:

72880fb: Call the filter again with runtime data this time
21a6f27: Fix name of filter-evaluation function
```
Originally added on 2015-06-19 11:46:03 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.12

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-07-13 17:13:16 +0000 UTC.

Changed State: Released
